docs/README.md

```diff
@@ -1,3 +1,3 @@
-The official documentation for this distribution can be found here:
+# The official documentation for this distribution can be found here:
```

Comment:
I'm not sure whether this actually should be a heading

Reply:
It's for formatting / visibility, mainly.